object-construction-checker/src/main/java/org/checkerframework/checker/objectconstruction/MustCallInvokedChecker.java

```diff
@@ -605,9 +731,9 @@ static String formatMissingMustCallMethods(List<String> mustCallVal) {
    */
   private static class BlockWithLocals {
     public final Block block;
-    public final Set<LocalVarWithTree> localSetInfo;
+    public final Set<Set<LocalVarWithTree>> localSetInfo;
```

Comment:
This field should have a type like `ImmutableSet<ImmutableSet<LocalVarWithTree>>`.  That will force us to be a lot more careful with mutation elsewhere.